packages/lib-utils/src/k8s/k8s-resource.ts

```diff
@@ -33,6 +33,7 @@ type K8sResourceListResult<TResource extends K8sResourceCommon> = {
   items: TResource[];
   metadata: {
     resourceVersion: string;
+    continue: string;
```

Comment:
Required for action logic here: https://github.com/openshift/dynamic-plugin-sdk/pull/37/files#diff-052548bf85cbe16601d8136f8af94f09dfa8c038d68d2affd1d9eb1a8b06e4ebR128